src/lib/select/select.ts

```diff
@@ -533,11 +552,14 @@ export class MdSelect implements AfterContentInit, OnDestroy, OnInit, ControlVal
    * @returns Option that has the corresponding value.
    */
   private _selectValue(value: any): MdOption {
-    let correspondingOption = this.options.find(option => option.value === value);
+    let optionsArray = this.options.toArray();
+    let correspondingOptionIndex = optionsArray.findIndex(option => option.value === value);
```

Comment:
We can't use `findIndex` inside of Google (Closure's type definition for array doesn't have it).

@crisbeto ping me when this one ready again